pkg/controller/deployment/util/deployment_util.go

```diff
@@ -857,7 +857,7 @@ func IsSaturated(deployment *apps.Deployment, rs *apps.ReplicaSet) bool {
 }
 
 // WaitForObservedDeployment polls for deployment to be updated so that deployment.Status.ObservedGeneration >= desiredGeneration.
-// Returns error if polling timesout.
+// Returns error if polling timeout.
```

Comment:
This does not seem like a typo to me.

Reply:
I think the correct is `times out`, ref. https://en.wiktionary.org/wiki/times_out#English